example/terraform.tfvars.example

```diff
@@ -7,3 +7,13 @@ system_bucket            = "my-internal-bucket"
 urs_client_id            = "abc123"
 urs_client_password      = "password"
 vpc_id                   = "vpc-123"
+
+# CloudWatch/Distribution Lambda logging
+log_api_gateway_to_cloudwatch = false # Defaults to false. Boolean switch to enable/disable logging of Api Gateway distribution traffic to CloudWatch.
+log_destination_arn           = "arn:aws:logs:us-east-1:123456789012:destination:LogDestination" # Remote logs destination to write logs.  (Must have log_api_gateway_to_cloudwatch set to true)
+
+# S3-replicator variables
+s3_replicator_source_bucket = "some-bucket"
```

Comment:
Remember to change this if you move to using an object-based config for s3 replicator stuff.